src/wavesurfer.js

```diff
@@ -963,6 +963,7 @@ export default class WaveSurfer extends util.Observer {
         }
 
         this.params.scrollParent = oldScrollParent;
+        this.fireEvent('seekfromto', {from:this.getCurrentTime(), to:progress});
```

Comment:
As an alternative, `{ from, to }` could be a second parameter to the existing `seek` event. WDYT?
Also, if `from` is a time in seconds, `to` should probably be as well a timestamp, not a float.
Please don't forget to document it.

Reply:
> a second parameter to the existing seek event.

Agreed.

Reply:
yea I was thinking of adding it to the existing one but older projects using the event might break

Reply:
we're planning on releasing 6.0 where it's ok to break some backwardcompat imo. as long as it's documented in the upgrade document..

Reply:
The chance of something breaking because of an extra argument at the end seems slim tbh.

Reply:
yes I'll return both as the actual timestamp